src/topology/separation.lean

```diff
@@ -554,3 +554,66 @@ begin
 end
 
 end normality
+
+/-- In a compact t2 space, the connected component of a point equals the intersection of all
+its clopen neighbourhoods. -/
+lemma connected_component_eq_Inter_clopen [t2_space α] [compact_space α] {x : α} :
+  connected_component x = ⋂ Z : {Z : set α // is_clopen Z ∧ x ∈ Z}, Z :=
+begin
+  apply eq_of_subset_of_subset,
+  { exact connected_component_subset_Inter_clopen},
+  { have hs : @is_closed _ _inst_1 (⋂ (Z : {Z : set α // is_clopen Z ∧ x ∈ Z}), ↑Z),
+    { exact is_closed_Inter (λ Z, Z.2.1.2) },
+    apply subset_connected_component,
```

Comment:
Again, among the two goals this line creates one is trivial, so you can write
```suggestion
    refine subset_connected_component _ (mem_Inter.2 (λ Z, Z.2.2)),
```
and then you're left with one single goal, and you can remove the accolade and unindent.